src/queries/articles/getAll.js

```diff
@@ -9,13 +9,14 @@ const { Op } = Sequelize;
  * @param {object} condition condition for query
  * @returns {object} Object representing the response returned
  */
-export default async (limit = 20, offset = 0) => {
+export default async (limit, offset) => {
   const condition = { status: { [Op.ne]: 'deleted' } };
```

Comment:
I do not understand this part, can you help me better understand.

Reply:
@luc-tuyishime **limit** is the number of article per page and **offset**  is used to determine the starting point to return articles rows from the database. So I removed those default values in the query to make sure that I am getting dynamic data from the controller.
Let me know things are clear or if you have a suggestion. Thanks

Reply:
Clear now thanks.